mandatory/1-fix-functions.js

```diff
@@ -10,20 +10,20 @@
 
 */
 
-function mood() {
-  let isHappy = true;
+function mood(mood) {
+  let isHappy=mood;
 
-  if (isHappy) {
+  if (isHappy===true) {
```

Comment:
You could remove`===true` and the code will still work.

Reply:
Changes made.thanks